composer/algorithms/seq_length_warmup/seq_length_warmup.py

```diff
@@ -184,6 +185,8 @@ def apply(self, event: Event, state: State, logger: Logger) -> Optional[int]:
             return
 
         assert state.dataloader is not None, "dataloader should be set on AFTER_DATALOADER"
+        if not isinstance(state.dataloader, torch.utils.data.DataLoader):
```

Comment:
This algorithm can work with FFCV dataloader, however, adding this check will make it fail as ffcv.Loader is not an instance of torch.utils.data.DataLoader.

Reply:
Good point; Replaced with a manual `getattr` check.